ql/termstructures/yield/ratehelpers.cpp

```diff
@@ -50,8 +50,10 @@ namespace QuantLib {
               case Futures::ASX:
                 QL_REQUIRE(ASX::isASXdate(date, false), date << " is not a valid ASX date");
                 break;
+              case Futures::Custom:
+                break;
               default:
-                QL_FAIL("unknown futures type (" << Integer(type) << ')');
+                QL_FAIL("unknown futures type (" << type << ')');
```

Comment:
Without the cast to Integer, this is going to trigger the QL_FAIL in `operator<<` — in this case the message will be the same, but in others the message there will override the original one.

Reply:
Removed QL_FAIL from operator<<, so now it is safe to use.